### PR TITLE
ARUHA-746: Added update of cleanup-time when changing retention time …

### PR DESCRIPTION
**Pull request by v-stepanov**
…for EventType

https://techjira.zalando.net/browse/ARUHA-746

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/EventTypeService.java [hunk 1]
```diff
@@ -269,7 +271,43 @@ public void delete(final String eventTypeName, final Client client)
         }
     }
 
-    private void updateRetentionTime(final String eventTypeName, final Long retentionTime)
+    private void updateEventTypeInDB(final EventType eventType, final Long newRetentionTime,
+                                     final Long oldRetentionTime) throws NakadiException {
+        final NakadiException exception = transactionTemplate.execute(action -> {
+            try {
+                eventTypeRepository.update(eventType);
+                updateTimelinesCleanup(eventType.getName(), newRetentionTime, oldRetentionTime);
```
Comment (adyach) [inline src/main/java/org/zalando/nakadi/service/EventTypeService.java]: @v-stepanov it is not clear how the event type cache will be updated? Maybe it make sense just to switch it with the previous line.

Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/service/EventTypeService.java]: Very good point 👍  I switched the order https://github.com/zalando/nakadi/pull/628/commits/7c4fd7a566ac2dc2a3698de9c096ed06fc239bee
Now the case will be updated with latest state of timelines.

**Comment by adyach** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Pull request merged**